### DISCUSSION TITLE
Parse error

### DISCUSSION THREAD

**Discussion by nametsego** [Cyfrin/foundry-full-course-cu#3083] [Q&A] [answered]
I need some help resolving this issue on the FundMe.sol codebase.

The issue originates from this particular function

  function getPrice() public view returns(uint256){
      AggregatorV3Interface priceFeed = AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306)
      (int256 price , uint256, uint256, uint256) = priceFeed.latestRoundData();
      return uint256(price*1e10);
    }

ParserError: Expected ',' but got identifier
--> FundMe.sol:15:15:
|
15 | (int256 price , uint256, uint256, uint256) = priceFeed.latestRoundData();
| ^^^^^

.

I need help please i'm stuck.

**Comment by satyaorz** [comment, marked as answer]
umm i think it should have been
```
AggregatorV3Interface(0x694AA1769357215DE4FAC081bf1f309aDC325306);
```

you missed a ";" ???

**Reply by EngrPips** [reply]
Great heads-up

**Reply by nametsego** [reply]
Resolved. Thank you so much! :)
